packages/rules-engine/src/processors/rulesEffectsProcessor/rulesEffectsProcessor.js

```diff
@@ -239,13 +239,17 @@ export function getRulesEffectsProcessor(
     }
 
     function processDisplayKeyValuePair(effect: ProgramRuleEffect): any {
+        const { data } = effect;
+        const dataToString = value => (typeof value == 'number' ? numberToString(value) : String(value));
+        const value = data !== undefined ? dataToString(data) : '';
```

Comment:
```suggestion
        const data = effect.data || '';
```

Reply:
Hold on, maybe `effect.data` could be `false`.. In that case this won't work.

Revised suggestion:
```suggestion
        const data = effect.data !== undefined ? effect.data : '';
```